sample/src/androidTest/java/com/schibsted/spain/barista/sample/FlakyRuleTest.java

```diff
@@ -51,7 +51,9 @@ public void someFlakyTest() throws Exception {
     }
   }
 
+  // WARNING: this test is flaky. Avoid running it at the C.I.
   @Test
+  @Ignore
```

Comment:
Nope. This test also crashes on localhost, so I prefer to keep it ignored on Travis AND in our computers. Newcomers doesn't know what's expected from this test, so it's strange to understand that a red is something randomly expected from this test. In addition, that's not related to passing the tests on Travis.